Remove verified twitter handle functionality

mvlabat opened the pull request.

bogdan commented:
I don't think we should remove that. The plan is to implement management in a different way: where UD is an authority to verify twitter handle without chainlink being man in the middle.

mvlabat commented:
My assumption was that since we removed `TwitterValidationOperator` and related smart contracts from the uns repo and removed the feature from the website, we should probably drop its support in the library. Do you think it'll have a similar interface and design so it's worth keeping?

bogdan commented:
Yes, we discussed it with dev tools and came up with the idea that fortunately new implementation can be fully compatible with the old one on the resolution side.
